synapse/config/tls.py

```diff
@@ -70,6 +71,50 @@ def read_config(self, config):
 
         self.tls_fingerprints = list(self._original_tls_fingerprints)
 
+        # Whether to verify certificates on outbound federation traffic
+        self.federation_verify_certificates = config.get(
+            "federation_verify_certificates", False,
+        )
+
+        # Whitelist of domains to not verify certificates for
+        federation_certificate_verification_whitelist = config.get(
+            "federation_certificate_verification_whitelist", [],
+        )
+
+        self.federation_certificate_verification_whitelist = None
+        if len(federation_certificate_verification_whitelist) > 0:
```

Comment:
in this case, there is no difference between an empty list and an absent list, so you can drop this condition and make the default `{}` (or the empty list), and save having to check for None elsewhere.